ci.cjs

```diff
@@ -0,0 +1,160 @@
+#! /usr/bin/env node
+
+const fs = require("fs").promises;
+const path = require("path");
+
+function printUsage() {
+  console.log(
+    `Usage: "ci check-consistency" or "ci bump-version %postfix%"`,
+  );
+}
+
+let postfix;
+const mode = process.argv[2];
+
+function validateArgs() {
+  switch (mode) {
+    case "get-version":
+      return true;
+
+    case "bump-version":
+      postfix = process.argv[3];
+      if (!postfix) {
+        printUsage();
+        process.exit();
+      }
+      return true;
+
+    case "":
+    case undefined:
+    case "check-consistency":
```

Comment:
what does that do?

Reply:
I don't really know. It is a copy paste script that was written long time ago by Pavel. I copy paste it and change it a little bit.

I plan to this logic to to github-actions that publishes pnpm snapshots in future...

Reply:
I checks if arguments are valid. checkConsistency is called in line ~147